src/wasm/wasm-s-parser.cpp

```diff
@@ -641,6 +641,16 @@ Type SExpressionWasmBuilder::stringToType(const char* str, bool allowError, bool
   throw ParseException("invalid wasm type");
 }
 
+Type SExpressionWasmBuilder::stringToLaneType(const char* str) {
+  if (strcmp(str, "i8x16") == 0) return i32;
+  if (strcmp(str, "i16x8") == 0) return i32;
+  if (strcmp(str, "i32x4") == 0) return i32;
+  if (strcmp(str, "i64x2") == 0) return i64;
+  if (strcmp(str, "f32x4") == 0) return f32;
+  if (strcmp(str, "f64x2") == 0) return f64;
+  return none;
```

Comment:
why is this not an error?

Reply:
Sorry, this fell off my radar somehow. I think any error here is handled in the calling code. It's a bit moot since there is only one caller, but this lets the caller decide what is an error and what is not without the complexity of an `allowError` argument.

Reply:
@kripken ping. Does that seem reasonable or should I change this? CI failures after the latest push were due to a timeout.

Reply:
Sounds reasonable, lgtm.